apps/web/pages/_app.tsx

```diff
@@ -70,8 +79,17 @@ export default withTRPC<AppRouter>({
           enabled: (opts) =>
             !!process.env.NEXT_PUBLIC_DEBUG || (opts.direction === "down" && opts.result instanceof Error),
         }),
-        httpBatchLink({
-          url: `/api/trpc`,
+        splitLink({
+          // check for context property `skipBatch`
+          condition: (op) => op.context.skipBatch === true,
+          // when condition is true, use normal request
+          true: httpLink({ url }),
+          // when condition is false, use batching
+          false: httpBatchLink({
+            url,
+            /** @link https://github.com/trpc/trpc/issues/2008 */
+            // maxBatchSize: 7
```

Comment:
Disabled until solved

Reply:
FYI `maxBatchSize` will disappear in the new major in favor of a property called `maxURILength`

Reply:
So I think you just don't want to use it.

Reply:
And what would be de difference in functionality?

Reply:
https://github.com/trpc/trpc/issues/1801#issue-1208401673
Add a new config option maxBatchURILength(?) to the httpBatchLink that will split the batch into smaller chunks before sending requests if the initial batch URI exceeds the limit.

Hey @zomars I don't know how it works now, but I'm thinking that since the trpc call it's just a long HTTP URI, it will limit the max size of a batch call, by the number of characters in the string but maybe @KATT or @jlalmes have more details on this

Reply:
^ Yes - that's exactly it @agustif!